temporalio/api/batch/v1/message_pb2.py

```diff
@@ -13,21 +13,6 @@
 _sym_db = _symbol_database.Default()
 
 
-from google.protobuf import timestamp_pb2 as google_dot_protobuf_dot_timestamp__pb2
-
-from temporalio.api.common.v1 import (
-    message_pb2 as temporal_dot_api_dot_common_dot_v1_dot_message__pb2,
-)
-from temporalio.api.dependencies.gogoproto import (
-    gogo_pb2 as dependencies_dot_gogoproto_dot_gogo__pb2,
-)
-from temporalio.api.enums.v1 import (
-    batch_operation_pb2 as temporal_dot_api_dot_enums_dot_v1_dot_batch__operation__pb2,
-)
-from temporalio.api.enums.v1 import (
-    reset_pb2 as temporal_dot_api_dot_enums_dot_v1_dot_reset__pb2,
-)
```

Comment:
These imports are intentionally side effects and should not be removed. I wonder if we need to make ruff ignore the entire `temporalio/api` folder